docs/concepts/advanced/rendering_js_css.md

```diff
@@ -1,24 +1,29 @@
-## JS and CSS output locations
```

Comment:
Rewrote the first half of the page. Some screen grabs:

<img width="1263" alt="Screenshot 2025-05-05 at 14 12 33" src="https://github.com/user-attachments/assets/05deedb9-f739-4dbe-9636-1f4f07993ffc" />

<img width="1220" alt="Screenshot 2025-05-05 at 14 12 39" src="https://github.com/user-attachments/assets/b2f58e5a-44dd-4589-8ec8-a48718ecab46" />

<img width="1205" alt="Screenshot 2025-05-05 at 14 12 48" src="https://github.com/user-attachments/assets/e75a0456-2131-4e93-81f3-b2071f585caf" />

<img width="1193" alt="Screenshot 2025-05-05 at 14 13 02" src="https://github.com/user-attachments/assets/0d0318e3-f81f-4e1e-ba59-99873c76aa75" />